### PR TITLE
[Cognitive Services] Update endpoint URL template for Content Moderator.

### PR DESCRIPTION
**Pull request by yangyuan**
Detail background and explainations in here: https://github.com/Azure/azure-rest-api-specs/issues/3489 Cognitive Services URL template (endpoint)

Please hold for swiftarrow11 or sanjeev3's review as well as my confirmation.

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md) document.
  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

### CODE REVIEW

File: specification/cognitiveservices/data-plane/ContentModerator/readme.nodejs.md [hunk 1]
```diff
@@ -8,7 +8,6 @@ nodejs:
   package-name: azure-cognitiveservices-contentmoderator
   package-version: 1.0.0-preview
   output-folder: $(node-sdks-folder)/lib/services/contentModerator
-  override-client-name: ContentModeratorAPIClient
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/ContentModerator/readme.nodejs.md]: Remove this to keep consistency

File: specification/cognitiveservices/data-plane/ContentModerator/stable/v1.0/ContentModerator.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "Content Moderator Client",
     "version": "1.0",
-    "description": "You use the API to scan your content as it is generated. Content Moderator then processes your content and sends the results along with relevant information either back to your systems or to the built-in review tool. You can use this information to take decisions e.g. take it down, send to human judge, etc.\r\n\r\nWhen using the API, images need to have a minimum of 128 pixels and a maximum file size of 4MB. \r\nText can be at most 1024 characters long. \r\nIf the content passed to the text API or the image API exceeds the size limits, the API will return an error code that informs about the issue.\r\n\r\nThis API is currently available in:\r\n\r\n* West US - westus.api.cognitive.microsoft.com\r\n* East US 2 - eastus2.api.cognitive.microsoft.com\r\n* West Central US - westcentralus.api.cognitive.microsoft.com\r\n* West Europe - westeurope.api.cognitive.microsoft.com\r\n* Southeast Asia - southeastasia.api.cognitive.microsoft.com ."
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/ContentModerator/stable/v1.0/ContentModerator.json]: Remove the outdated region list.

**Comment by jhendrixMSFT** [comment]
@yangyuan are we still waiting for other reviewers?

**Comment by yangyuan** [comment]
Hi @jhendrixMSFT. I've updated some SDKs, but heard some negative feedback (also found an small issue in auto-generated documents).
So I decide to hold this PR for a few days and collect feedbacks in case of back/forth changes.

**Comment by sanjeev3** [comment]
@yangyuan @jhendrixMSFT, I am good with these. @swiftarrow11, how about you?

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
There are some model validation failures that need to be addressed, please see the travis [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/408262332).

**Comment by yangyuan** [comment]
hi @jhendrixMSFT, I've fixed the model validation error.

It shows that Linter Diff complaints some operations missing x-ms-examples.
How about we create an issue and deal with it later? two reasons I recommend that.
1. Its some legacy issues not related to this change.
2. The examples should be provided by Content Moderator team.

**Comment by jhendrixMSFT** [comment]
@bsiegel I understand you're working on this new linter diff in CI?  I don't think it should be failing for this PR as it doesn't add any new operations.

**Comment by jhendrixMSFT** [comment]
@veronicagg do you know about this new linter behavior?

**Comment by veronicagg** [comment]
@jhendrixMSFT I asked @bsiegel about this in the morning. @bsiegel I think this is another case for the issue you fixed earlier today, in which case this branch would need to be rebased to pick up Brandon's fix. @bsiegel could you check other PRs that may be opened and got affected by it?

**Comment by yangyuan** [comment]
@jhendrixMSFT I have rebase the branch and now all passes. :)

**Review by jhendrixMSFT — APPROVED** [review]
There are some model validation failures that need to be addressed, please see the travis [log](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/408262332).

**Pull request merged**